diff --git a/recording-daemon/Makefile b/recording-daemon/Makefile
index 200636e..6b0ca7a 100644
--- a/recording-daemon/Makefile
+++ b/recording-daemon/Makefile
@@ -10,7 +10,6 @@ CFLAGS+=	$(shell pkg-config --cflags libavformat)
 CFLAGS+=	$(shell pkg-config --cflags libavutil)
 CFLAGS+=	$(shell pkg-config --cflags libswresample)
 CFLAGS+=	$(shell pkg-config --cflags libavfilter)
-CFLAGS+=	$(shell mysql_config --cflags)
 CFLAGS+=	$(shell pkg-config --cflags openssl)
 
 LDFLAGS=	-lm
@@ -21,11 +20,10 @@ LDFLAGS+=	$(shell pkg-config --libs libavformat)
 LDFLAGS+=	$(shell pkg-config --libs libavutil)
 LDFLAGS+=	$(shell pkg-config --libs libswresample)
 LDFLAGS+=	$(shell pkg-config --libs libavfilter)
-LDFLAGS+=	$(shell mysql_config --libs)
 LDFLAGS+=	$(shell pkg-config --libs openssl)
 
 SRCS=		epoll.c garbage.c inotify.c main.c metafile.c stream.c recaux.c packet.c \
-		decoder.c output.c mix.c db.c log.c forward.c
+		decoder.c output.c mix.c log.c forward.c
 LIBSRCS=	loglib.c auxlib.c rtplib.c codeclib.c resample.c str.c
 OBJS=		$(SRCS:.c=.o) $(LIBSRCS:.c=.o)
 
diff --git a/recording-daemon/epoll.c b/recording-daemon/epoll.c
index 68e86b9..0332bea 100644
--- a/recording-daemon/epoll.c
+++ b/recording-daemon/epoll.c
@@ -3,7 +3,6 @@
 #include <glib.h>
 #include <pthread.h>
 #include <unistd.h>
-#include <mysql.h>
 #include "log.h"
 #include "main.h"
 #include "garbage.h"
@@ -33,7 +32,6 @@ void epoll_del(int fd) {
 
 
 static void poller_thread_end(void *ptr) {
-	mysql_thread_end();
 }
 
 
@@ -43,7 +41,6 @@ void *poller_thread(void *ptr) {
 
 	dbg("poller thread %u running", me_num);
 
-	mysql_thread_init();
 
 	pthread_cleanup_push(poller_thread_end, NULL);
 
diff --git a/recording-daemon/main.c b/recording-daemon/main.c
index ef5b2ff..9e90123 100644
--- a/recording-daemon/main.c
+++ b/recording-daemon/main.c
@@ -12,7 +12,6 @@
 #include <libavutil/log.h>
 #include <sys/stat.h>
 #include <sys/types.h>
-#include <mysql.h>
 #include "log.h"
 #include "epoll.h"
 #include "inotify.h"
@@ -73,7 +72,6 @@ static void setup(void) {
 				die_errno("Failed to create output dir '%s'");
 		}
 	}
-	mysql_library_init(0, NULL, NULL);
 	signals();
 	metafile_setup();
 	epoll_setup();
@@ -129,7 +127,6 @@ static void cleanup(void) {
 	metafile_cleanup();
 	inotify_cleanup();
 	epoll_cleanup();
-	mysql_library_end();
 }
 
 
diff --git a/recording-daemon/metafile.c b/recording-daemon/metafile.c
index 64ce12f..233594d 100644
--- a/recording-daemon/metafile.c
+++ b/recording-daemon/metafile.c
@@ -57,13 +57,11 @@ static void meta_destroy(metafile_t *mf) {
 		close(mf->forward_fd);
 		mf->forward_fd = -1;
 	}
-	db_close_call(mf);
 }
 
 
 // mf is locked
 static void meta_stream_interface(metafile_t *mf, unsigned long snum, char *content) {
-	db_do_call(mf);
 	if (output_enabled) {
 		pthread_mutex_lock(&mf->mix_lock);
 		if (!mf->mix && output_mixed) {
@@ -71,7 +69,6 @@ static void meta_stream_interface(metafile_t *mf, unsigned long snum, char *cont
 			snprintf(buf, sizeof(buf), "%s-mix", mf->parent);
 			mf->mix_out = output_new(output_dir, buf);
 			mf->mix = mix_new();
-			db_do_stream(mf, mf->mix_out, "mixed", 0, 0);
 		}
 		pthread_mutex_unlock(&mf->mix_lock);
 	}
@@ -108,7 +105,6 @@ static void meta_rtp_payload_type(metafile_t *mf, unsigned long mnum, unsigned i
 // mf is locked
 static void meta_metadata(metafile_t *mf, char *content) {
 	mf->metadata = g_string_chunk_insert(mf->gsc, content);
-	db_do_call(mf);
 	if (forward_to)
 		start_forwarding_capture(mf, content);
 }
diff --git a/recording-daemon/output.c b/recording-daemon/output.c
index a901cc7..24760f9 100644
--- a/recording-daemon/output.c
+++ b/recording-daemon/output.c
@@ -121,7 +121,6 @@ got_fn:
 		goto err;
 
 done:
-	db_config_stream(output);
 	return 0;
 
 err:
@@ -159,7 +158,6 @@ void output_close(output_t *output) {
 	if (!output)
 		return;
 	output_shutdown(output);
-	db_close_stream(output);
 	encoder_free(output->encoder);
 	g_slice_free1(sizeof(*output), output);
 }
diff --git a/recording-daemon/packet.c b/recording-daemon/packet.c
index 5c9baed..ffead5e 100644
--- a/recording-daemon/packet.c
+++ b/recording-daemon/packet.c
@@ -53,7 +53,6 @@ static ssrc_t *ssrc_get(stream_t *stream, unsigned long ssrc) {
 	snprintf(buf, sizeof(buf), "%s-%08lx", mf->parent, ssrc);
 	if (output_single) {
 		ret->output = output_new(output_dir, buf);
-		db_do_stream(mf, ret->output, "single", stream->id, ssrc);
 	}
 
 	g_hash_table_insert(mf->ssrc_hash, GUINT_TO_POINTER(ssrc), ret);
